### PR TITLE
code fixes 1

### PR DESCRIPTION
**Pull request by Devesh21700Kumar**
1).restructured props in pages/home
2).relative imports from directories external to /src are not allowed in modern react versions so fixed them to adapt with latest versions
3).made Avatar a functional component

for issue #3

### CODE REVIEW

**Review by tewarig — COMMENTED** [review]
well done... please mention the issue as well this pr is solving. I think it is #3

**Comment by Devesh21700Kumar** [comment]
> well done... please mention the issue as well this pr is solving. I think it is #3

thanks.. yeah i have mentioned it now

File: src/content/Avatar/index.jsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import React, { lazy, Component, Suspense } from "react";
 import Playground from "./avatar.playground.jsx";
-import { importMDX } from "mdx.macro";
+import importMDX from "mdx.macro";
 import PropDrawer from "../../components/PropDrawer";
 
 const Content = lazy(() => importMDX("./avatar.mdx"));
-class App extends Component {
-  render() {
+export default function App(){
```
Comment (pranshuchittora) [inline src/content/Avatar/index.jsx:7]: Can you convert this to a named function. Default export of anonymous funcs is not recommended 
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-anonymous-default-export.md

File: src/pages/home/index.jsx [hunk 2]
```diff
@@ -83,7 +83,7 @@ function HomePage() {
 
 export default HomePage;
 
-const FeatureCard = (props) => {
+const FeatureCard = ({banner,title,content}) => {
```
Comment (pranshuchittora) [inline src/pages/home/index.jsx:86]: Not required

Comment (pranshuchittora) [reply src/pages/home/index.jsx:86]: This will be fixed after TS migration of this repo

File: src/content/Badge/index.jsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React, { lazy, Suspense } from "react";
-import { importMDX } from "mdx.macro";
+import importMDX from "mdx.macro";
```
Comment (pranshuchittora) [inline src/content/Badge/index.jsx:2]: Incorrect, breaking the build.
```
react_devtools_backend.js:2430 ReferenceError: importMDX is not defined
    at index.jsx:7
    at 2.b56122d9.chunk.js:sourcemap:2
    at ys (2.b56122d9.chunk.js:sourcemap:2)
    at ul (2.b56122d9.chunk.js:sourcemap:2)
    at sl (2.b56122d9.chunk.js:sourcemap:2)
    at Zs (2.b56122d9.chunk.js:sourcemap:2)
    at 2.b56122d9.chunk.js:sourcemap:2
    at t.unstable_runWithPriority (2.b56122d9.chunk.js:sourcemap:2)
    at Ui (2.b56122d9.chunk.js:sourcemap:2)
    at Gi (2.b56122d9.chunk.js:sourcemap:2)
```

**Review by pranshuchittora — CHANGES_REQUESTED** [review]

**Comment by pranshuchittora** [comment]
> Deploy preview for _rne-playground_ ready!
> 
> Built with commit [b16bc2d](https://github.com/react-native-elements/playground/commit/b16bc2d1acac988371f69f5191796237bd86d47d)
> 
> https://deploy-preview-31--rne-playground.netlify.app

@Devesh21700Kumar pls check the build from your changes.
1. Go to any component
2. Click the Prop Explorer at the bottom right
3. Check the console for the respective err

File: src/content/Avatar/index.jsx (hunk 1 above)
Comment (Devesh21700Kumar) [reply src/content/Avatar/index.jsx:7]: sure

File: src/pages/home/index.jsx (hunk 2 above)
Comment (Devesh21700Kumar) [reply src/pages/home/index.jsx:86]: ok

File: src/content/Badge/index.jsx (hunk 3 above)
Comment (Devesh21700Kumar) [reply src/content/Badge/index.jsx:2]: is importMDX defined for other files?

**Comment by Devesh21700Kumar** [comment]
@pranshuchittora Hey, Thanks for the review.. I've fixed the Build errors and have made the export named instead of default.. please do check and merge if it is not giving any errors

File: src/pages/home/index.jsx (hunk 2 above)
Comment (pranshuchittora) [reply src/pages/home/index.jsx:86]: Undo these changes as requested

Comment (Devesh21700Kumar) [reply src/pages/home/index.jsx:86]: ok

Comment (Devesh21700Kumar) [reply src/pages/home/index.jsx]: yeah @pranshuchittora reverted them. thanks

**Comment by Devesh21700Kumar** [comment]
@pranshuchittora I have reverted them now

**Comment by pranshuchittora** [comment]
Pls resolve the conflicts

**Pull request closed without merging**